Add color for multiple missing elements

waruhachi opened the pull request:
Added the appropriate colors for multiple new elements

mvllow commented:
Not sure if intentional or not but the font has been removed. I personally don't mind leaving it out, though.

Would you mind sharing screenshots of the changes?

waruhachi commented:
I removed it intentionally as I don't see any significance in its relationship to the theme. I think it can be left to the individual user to use or not use a custom font alongside the theme.

Along with some more elements covered, the code has also been organized to make sure both dark and light modes are consistent. I have included screenshots reflecting the changes made below. This doesn't show all the places the changes can be seen but a majority of them.

<details>

<summary>Images</summary>

### Active Now

- Added card background

Before             |  After
:-------------------------:|:-------------------------:
![CleanShot 2024-07-24 at 20 04 45@2x](https://github.com/user-attachments/assets/0443712a-025e-4b7a-8d2e-4c925beaa249) | ![CleanShot 2024-07-24 at 20 04 53@2x](https://github.com/user-attachments/assets/68eda721-38da-45da-961d-58bb01be0cc5)
![CleanShot 2024-07-24 at 20 17 09@2x](https://github.com/user-attachments/assets/3684ed7b-7965-41f3-a96d-f3f1974bb885) | ![CleanShot 2024-07-24 at 20 16 59@2x](https://github.com/user-attachments/assets/3bd867fb-20ea-48b6-9147-bda19cfa18b3)


### Profile Popout

- Added background to profiles and elements
- Removed background and border for badges

Before             |  After
:-------------------------:|:-------------------------:
![CleanShot 2024-07-24 at 20 13 53@2x](https://github.com/user-attachments/assets/0a80eff8-583b-41f7-908d-8832aa85472a) | ![CleanShot 2024-07-24 at 20 14 03@2x](https://github.com/user-attachments/assets/d85639f6-1f1d-4841-bb98-4c232e27f354)
![CleanShot 2024-07-24 at 20 15 29@2x](https://github.com/user-attachments/assets/0208f765-4e9b-43eb-9aa8-ff60f338599d) | ![CleanShot 2024-07-24 at 20 15 18@2x](https://github.com/user-attachments/assets/abb9bbc6-0532-45ce-b526-bdf3295ede5f)
![CleanShot 2024-07-24 at 20 28 02@2x](https://github.com/user-attachments/assets/9bd77bd0-edcd-4577-9d6e-3802b2c9a116) | ![CleanShot 2024-07-24 at 20 27 49@2x](https://github.com/user-attachments/assets/23f6a592-36ef-450a-a0a7-d66d0c96ad88)
![CleanShot 2024-07-24 at 20 28 53@2x](https://github.com/user-attachments/assets/5756077c-1814-4a2f-927b-1cc639345689) | ![CleanShot 2024-07-24 at 20 28 53@2x](https://github.com/user-attachments/assets/8554a54b-5c10-47cc-a7d7-55ce334a0047)
![CleanShot 2024-07-24 at 20 22 14@2x](https://github.com/user-attachments/assets/aa995bb4-800f-48e2-8282-39a50fb8646c) | ![CleanShot 2024-07-24 at 20 22 03@2x](https://github.com/user-attachments/assets/07b2b5ea-959b-4bf9-b312-7506d0d242d0)
![CleanShot 2024-07-24 at 20 24 32@2x](https://github.com/user-attachments/assets/bd4e6c00-84d8-4cdf-aa23-0505c00ec828) | ![CleanShot 2024-07-24 at 20 24 41@2x](https://github.com/user-attachments/assets/0d840e87-5634-4ed7-a4b8-196d4110007c)

### Modals

- Added background, hover, and select colors

Before             |  After
:-------------------------:|:-------------------------:
![CleanShot 2024-07-24 at 20 36 41@2x](https://github.com/user-attachments/assets/1971b1f1-e056-48ff-9312-63a95f50ff15) | ![CleanShot 2024-07-24 at 20 36 30@2x](https://github.com/user-attachments/assets/1300b6ed-b4d0-4732-8a4a-3585f26b64d0)
![CleanShot 2024-07-24 at 20 39 57@2x](https://github.com/user-attachments/assets/8a10fe85-60da-492c-b9c8-f600687ed1f6) | ![CleanShot 2024-07-24 at 20 40 06@2x](https://github.com/user-attachments/assets/a3ab722b-26a0-41a9-9bd7-d74f8e62d148)
![CleanShot 2024-07-24 at 21 31 52@2x](https://github.com/user-attachments/assets/98b99c72-25a6-412c-8544-8630071e6b8f) | ![CleanShot 2024-07-24 at 21 31 35@2x](https://github.com/user-attachments/assets/287bc647-6bc3-4ad9-9d2e-4bfa68b87734)
![CleanShot 2024-07-24 at 21 32 30@2x](https://github.com/user-attachments/assets/e49ada88-a48d-4f62-8b8b-b9b7f4f707e0) | ![CleanShot 2024-07-24 at 21 32 45@2x](https://github.com/user-attachments/assets/2f580511-9bc1-438b-9270-c7080bd3e4a5)

### Miscellaneous

- Blend profile quick actions into the sidebar
- Match scrollbar colors

Before             |  After
:-------------------------:|:-------------------------:
![CleanShot 2024-07-24 at 21 37 12@2x](https://github.com/user-attachments/assets/9599c96e-a4d1-44a7-845d-2c6f73525e2a) | ![CleanShot 2024-07-24 at 21 37 03@2x](https://github.com/user-attachments/assets/a9a18b41-68a8-4c5b-990e-dd661316a6c7)
![CleanShot 2024-07-24 at 21 38 23@2x](https://github.com/user-attachments/assets/32e4c2b9-b59b-45c5-907b-cd2a44b629b5) | ![CleanShot 2024-07-24 at 21 38 36@2x](https://github.com/user-attachments/assets/c742095d-2f7b-47de-98f9-71f2b961ad7d)
![CleanShot 2024-07-24 at 21 41 11@2x](https://github.com/user-attachments/assets/c64cb420-5bd3-4df0-b2fb-a3c668aeb893) | ![CleanShot 2024-07-24 at 21 41 02@2x](https://github.com/user-attachments/assets/3e9626c7-3a30-460c-94d4-5cd9011180d4)
![CleanShot 2024-07-24 at 21 42 17@2x](https://github.com/user-attachments/assets/bf820f99-2bb1-4628-9405-4a4f73a17a1f) | ![CleanShot 2024-07-24 at 21 42 09@2x](https://github.com/user-attachments/assets/914d59e2-5ac8-4f88-9b5d-69a49ba95e27)

</details>

mvllow commented:
Thank you for the before & afters, everything looks great!
